_posts/2023-01-18-Introducing-Identity.markdown

```diff
@@ -0,0 +1,73 @@
+
+### Plugin and extension restrictions
+
+Just as downloading an app to your smartphone includes restrictions intended to prevent surprises and risk, downloading a plugin to OpenSearch should be just as safe and predictable a process. To create this kind of experience for all actions involving plugins and extensions, we intend to introduce certain restrictions that will allow administrators to handle tasks confidently and avoid having to second-guess whether an action might involve any risk.
+
+To ensure that these kinds of limitations are enforced, we plan to leverage the same structures used to grant users access and map permissions for all plugin and extension management activities. As a result, we will reduce the complexity and risk involved in managing these tasks.
+
+### Background task permissions
+
+There are many tasks that run in the background of an OpenSearch cluster, such as deletion of old indexes and generation of monitoring data and reports. When these tasks run, each should have only the minimum number of permissions needed to protect against task errors and any unintended results and their consequences.
+
+Associating identity to tasks will ensure that permissions applied to these tasks are well defined and effective at preventing errors and unexpected results.
+
+### Security isolation 
+
+Following the principle of least privilege, using a minimum number of permissions to run tasks is key to preventing execution errors or software flaws from impacting the stability of an OpenSearch cluster. By adding features that can protect all of the separate elements in OpenSearch by isolating the potential impact a problem in one element may have on another element. This will reduce the spread of errors and keep them from affecting other areas of OpenSearch.
+
+## Get involved
+
+These features present a significant departure from the current security model. It does so by providing features that make it easier for OpenSearch developers to access and build assets and for administrators to carry out tasks in a stable environment. Over the last several months, we’ve been busy defining new features and building tools to prepare for the integration of Security into core OpenSearch. We’ve struck out on this path in an effort to make OpenSearch security features more efficient, more reliable, and simply easier and more enjoyable to work with.
+
+And since we operate in the open-source community, we’d like to learn about your ideas and benefit from your contributions as we make progress.
+
+Watch for further blog posts on specific identity and access control features, and join us for community meetings. Furthermore, you can stay informed of development by visiting the following resources in the OpenSearch repository:
```

Comment:
I still think it would be a good idea to provide links to where the reader can submit their ideas or join a community meeting.

Reply:
Added a feedback link